stylesheets/_component.tables.scss

```diff
@@ -1,115 +1,148 @@
-table {
-    background-color: $table-bg;
-    border-collapse: separate;
-}
+.table {
+    margin-bottom: 24px;
 
-th {
-    text-align: left;
-}
+    a {
+        border-bottom: 1px dotted color(text, link);
+        text-decoration: none;
+    }
 
-// Columns
-table {
-    th {
-        background-color: darken($white, 5%);
-        font-family: $font-family-regular;
-        font-weight: normal;
+    tbody > tr:hover > td {
+        background-color: color(background, light);
     }
-}
 
-.table {
-    margin-top: -6px;
-    margin-bottom: $line-height-base + 5;
-
-    width: 97.7%;
-
-    // Cells
-    thead,
-    tbody,
-    tfoot {
-        > tr {
-            > th,
-            > td {
-                padding: 0 5px;
-                line-height: $line-height-base * 2 - 1; // account for border
-                vertical-align: top;
-                border-top: 1px solid $table-border-color;
-            }
+    tr {
+        vertical-align: top;
+
+        a,
+        a:link {
+            border-bottom-color: color(link, dark);
+            color: color(link, dark);
         }
     }
-    // Bottom align for column headings
-    thead > tr > th {
-        vertical-align: bottom;
-    }
-    // Remove top border from thead by default
-    caption + thead,
-    colgroup + thead,
-    thead:first-child {
-        tr:first-child {
-            th,
-            td {
-                border-top: 1px solid transparent;
-            }
+
+    tr.selected td,
+    tr.is-selected td {
+        background-color: color(background, selected);
+
+        a,
+        a:link {
```

Comment:
Change to `a { &, &:link { ... } }` instead, like how line 35 (below) is laid out?